### PR TITLE
Major doc edits and repo cleanup

### PR DESCRIPTION
**Pull request by johnwcallahan**
## Description

⚠️ Don't be tempted to review this one lightly. ⚠️ 

My main goal was to consolidate documentation. We've accumulated a lot of content over the years and honestly a lot of it is outdated. I also removed red herrings like **everything related to Docker** (since we don't actually use it).

I deleted A LOT of stuff. I tried to ensure everything that I deleted was either a) covered more succinctly somewhere else, or b) outdated. The goal is to lower the barrier of entry to get going with the repo.

Highlights:

- Rewrite GETTING_STARTED.md. 
- Rewrite CONTRIBUTING.md. 
- Delete a lot of guides.
- Move most existing guides to a new directory, `/docs`.
- Remove Docker integration
- Update pull request template.

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -1,92 +1,57 @@
 # Getting Started
 
-Set up and run Linode's UI packages locally.
+1. Fork this repository.
+2. Clone your fork to your local machine.
+3. Go to [cloud.linode.com/profile/clients](https://cloud.linode.com/profile/clients) and click "Add an OAuth App".
+4. Enter a label and set the callback URL to `http://localhost:3000/oauth/callback`.
+5. After your OAuth App has been created, copy the ID (not the secret).
+6. In `packages/manager` , copy the contents of `.env.example` and paste them into a new file called `.env`.
```
Comment (tiffwong) [inline GETTING_STARTED.md:8]: There seems to be an extra space between `packages/manager` and ", copy..."

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,22 +1,9 @@
 ## Description
 
-Please start the pull request title with the jira ticket corresponding to the work if applicable. Please include a short summary of the feature added, the change, or issue fixed.
+**What does this PR do?**
```
Comment (tiffwong) [inline PULL_REQUEST_TEMPLATE.md:3]: So glad this is finally updated

File: packages/manager/.env.example [hunk 3]
```diff
@@ -32,18 +31,26 @@ REACT_APP_API_ROOT='https://api.linode.com/v4'
...
 # REACT_APP_API_MAX_PAGE_SIZE=
 
+# The Statuspage ID for displaying incidents and maintenance:
+# REACT_APP_STATUS_PAGE_ID=
+
+# The PayPal environment to use ("production" or "sandbox"):
+# REACT_APP_PAYPAL_ENV=
+
+# Set to "true" to disable /event request throttling (useful for E2E tests):
+# REACT_APP_DISABLE_EVENT_THROTTLE
```
Comment (tiffwong) [inline packages/manager/.env.example:41]: Should there be an "=" sign after "THROTTLE"?

**Review by tiffwong — APPROVED** [review]
These are great, thanks for cleaning this up!

**Review by ndukatz — APPROVED** [review]
This is so much better. I can work on adding the details for e2e tests in another PR

File: TESTING.md [hunk 4]
```diff
@@ -1,10 +1,8 @@
-TESTING.md
-
-# Testing Linode Manager
+# Testing Cloud Manager
 
 ## Unit Tests
 
-The unit tests for Linode Cloud Manager are written in Typescript using the [Jest](https://facebook.github.io/jest/) testing framework. Unit tests end with the `.test.tsx` file extension and can be found throughout the codebase.
+The unit tests for Cloud Manager are written in Typescript using the [Jest](https://facebook.github.io/jest/) testing framework. Unit tests end with the `.test.tsx` file extension and can be found throughout the codebase.
```
Comment (DevDW) [inline TESTING.md:5]: "...end with either the `.test.ts` or `.test.tsx` extensions (depending on whether JSX is rendered or not -- in the former case, it is not)..."

**Review by DevDW — APPROVED** [review]

File: GETTING_STARTED.md [hunk 5]
```diff
@@ -1,92 +1,57 @@
...
-To learn more about the available commands read [COMMANDS](./COMMANDS.md)
+```
 
-## Okay. I've got my development server running. So how do I contribute?
+9. Install the latest version of Yarn:
 
-Please see our [contributing](./CONTRIBUTING.md) and [code conventions](./CODE_CONVENTIONS.md) guides for instructions on how to get started with contributing to this project.
+```bash
+$ npm install --global yarn --upgrade
+# 1.22.4
```
Comment (Jskobos) [inline GETTING_STARTED.md]: 1.22.10 now 🤷

File: RELEASING.md [hunk 6]
```diff
@@ -1,95 +0,0 @@
...
-whether `staging` has a git tag commit. If it does
-
-1. Run `git checkout staging && git pull && git checkout testing && git rebase -i staging`
-2. Then proceed to merge testing into staging
-
-Finally, start again at step 11 of the above instructions
-
-At the end of the release process, `master` branch will be merged back into `develop`, so don't fear that you also need to apply your changes to `develop`.
-
-## Generating the changelog
```
Comment (Jskobos) [inline RELEASING.md:72]: Is the changelog usage documented anywhere else (internally)?

**Review by Jskobos — APPROVED** [review]
Looks good.

1. Maybe getting_started should live in the docs/ directory?
2. I don't think there are any substantial changes to the getting started steps, but it couldn't hurt to let Support volunteers try them from scratch to make sure they still work for people coming from the outside.

File: RELEASING.md (hunk 6 above)
Comment (johnwcallahan) [reply RELEASING.md:72]: Yes it is, in our internal Deploying Cloud Manager guide.

**Comment by johnwcallahan** [comment]
Thanks for the feedback, all. @Jskobos I moved GETTING_STARTED and CONTRIBUTING into /docs as well.

**Pull request merged**